config/index.js

```diff
@@ -0,0 +1,179 @@
+var rp = require('request-promise');
+var fs = require('fs');
+var google = require('googleapis');
+
+var ENVIRONMENT = process.env.ENVIRONMENT || 'dev'
+var PROJECT_ID = `pi-ostelco-${ENVIRONMENT}`;
+var HOST = 'https://firebaseremoteconfig.googleapis.com';
+var PATH = '/v1/projects/' + PROJECT_ID + '/remoteConfig';
+var SCOPES = ['https://www.googleapis.com/auth/firebase.remoteconfig'];
+
+/**
+ * Get a valid access token.
+ */
+// [START retrieve_access_token]
```

Comment:
I'm unfamiliar with .js coding conventions.   Does the "// [START ...]" pattern have some special significance ?

Reply:
I'm also unfamiliar with this type of syntax, it was part of the template and I didn't want to heavily modify the quickstart template provided by firebase.